utilities/game.py

```diff
@@ -146,3 +185,35 @@ def get_staged_pieces(alliance: wpilib.DriverStation.Alliance) -> list[Translati
         return STAGED_PIECES_BLUE
     else:
         return STAGED_PIECES_RED
+
+
+def is_red() -> bool:
+    return get_team() == wpilib.DriverStation.Alliance.kRed
+
+
+def get_team() -> wpilib.DriverStation.Alliance:
+    return wpilib.DriverStation.getAlliance()
+
+
+def get_cone_pickup(targeting_left: bool) -> tuple[Pose2d, Rotation2d]:
+    # if we want the substation to be as if we are on the red alliance
+    red_side = is_red() != swap_substation
```

Comment:
This will always be True, as a tunable instance is never equal to a boolean…

Reply:
I think more broadly that file needs a really good refactor - it's not very clear how it all works, and there are a lot of arguments to functions that probably aren't necessary.